process_labels/loading_funcs/tanzania_ecaas.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+import json
+import geopandas
+import pandas as pd
+from datetime import datetime
+from shapely.geometry import Point
+from cropharvest.columns import RequiredColumns, NullableColumns
+from cropharvest.config import EXPORT_END_MONTH, EXPORT_END_DAY
+
+# from ..utils import DATASET_PATH # ImportError: attempted relative import with no known parent package
+
+from typing import List
+
+LABEL_TO_CLASSIFICATION = {
+    "rice": "cereals",
+    "maize": "cereals",
+}
+
+
+def load_tanzania_ecaas():
```

Comment:
Could you add this to the `tanzania.py` file? This allows us to have a single file per country